maven/action.yml

```diff
@@ -39,6 +39,9 @@ inputs:
       This value will not be used if `NEXUS_PASSWORD` is already set as an env var.
     default: nexus-password
     required: false
+  working-dir:
```

Comment:
I'd prefer that we use `working-directory` to match the name used by the built-in `run` step.

Reply:
updated ✅